### PR TITLE
Added UT of syncController

### PR DESCRIPTION
**Pull request by vincentgoat**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
Added UT of syncController

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by vincentgoat** [comment]
Fix sub-items of https://github.com/kubeedge/kubeedge/issues/4327

File: cloud/pkg/synccontroller/synccontroller_test.go [hunk 1]
```diff
@@ -0,0 +1,199 @@
+package synccontroller
```
Comment (gy95) [inline cloud/pkg/synccontroller/synccontroller_test.go:1]: add a copyright

**Review by wackxu — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**